doc/classes/AudioStreamSample.xml

```diff
@@ -1,10 +1,12 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="AudioStreamSample" inherits="AudioStream" category="Core" version="3.2">
 	<brief_description>
-		Plays audio.
+		Audio stream driver for WAV files and non-realtime dynamically generated PCM audio data.
 	</brief_description>
 	<description>
-		Plays audio, can loop.
+		Audio stream driver for WAV files and non-realtime dynamically generated PCM audio data.
+		
```

Comment:
Don't add empty lines in the XML, each line break creates a new paragraph.

Reply:
Removed empty line in in https://github.com/follower/godot/commit/c0bd96e0e45567ae2bc36cc9963dd28d0613d7fa